src/sonic-yang-models/tests/yang_model_tests/tests_config/syslog.json

```diff
@@ -1,4 +1,75 @@
 {
+    "SYSLOG_SERVER_VALID": {
+        "sonic-syslog:sonic-syslog": {
+            "sonic-syslog:SYSLOG_SERVER": {
+                "SYSLOG_SERVER_LIST": [
+                    {
+                        "server_address": "100.0.0.1",
+                        "source": "1.1.1.1",
+                        "port": "514",
+                        "vrf": "default"
+                    },
+                    {
+                        "server_address": "200.0.0.1",
+                        "source": "1.1.1.1",
+                        "port": "514",
+                        "vrf": "mgmt"
```

Comment:
Suggest to have a validation for mgmt VRF ' (/mvrf:sonic-mgmt_vrf/mvrf:MGMT_VRF_CONFIG/mvrf:vrf_global/mvrf:mgmtVrfEnabled = 'true')' as well.

Reply:
@venkatmahalingam not possible since it goes against the feature design: with the approach suggested won't be able to use a definition for config generator (template renderer)

Reply:
YANG should do all possible validations (including custom validations if required), delegating any validation to CLI is not a good idea.

Reply:
@venkatmahalingam done